internal/kafka/test/integration/data_plane_cluster_test.go

```diff
@@ -613,7 +625,9 @@ func TestDataPlaneCluster_WhenReportedStrimziVersionsIsEmptyAndClusterStrimziVer
 		t.Fatalf("Cluster not found")
```

Comment:
Not part of this PR, but I believe this should be changed to 
```go
Expect(testDataPlaneclusterID).ToNot(BeEqual(""));
```

Same should be done for the other check (`if err != nil` ==> `Expect(err).ToNot(HaveOccurred())`

Reply:
I changed occurrences of checking errors like this `if err != nil` and failing with `t.Fatal` and `t.Error` to use gomega expect assertion. I only changed it for the data plane cluster test though but saw several instances where we do this in other test files. 

I think I can create a follow up PR to refactor that to use gomega instead, wdyt?

Reply:
Agree